command/maintenance_window_list.go

```diff
@@ -23,5 +38,46 @@ func (c *MaintenanceWindowList) Synopsis() string {
 }
 
 func (c *MaintenanceWindowList) Run(args []string) int {
+	var includes, serviceIDs, teamIDs []string
+	var query, filter string
+	flags := c.Meta.FlagSet("maintenance-window list")
+	flags.Usage = func() { fmt.Println(c.Help())}
```

Comment:
This also looks like `go fmt` wasn't ran on save, and so do the rest of the files.